src/chapters/objects-and-math/codingwithobjects.rst

```diff
@@ -0,0 +1,163 @@
+Coding With Objects
+===================
+
+Booleans and Objects
+--------------------
+
+Objects are not stored by their properties or by value, but by reference.
+Storing something by reference means that it is stored based on its location in memory.
+This can lead to some confusion when comparing objects.
+
+.. admonition:: Example
+
+   Let's see how this affects our zoo software! Surely, the zoo has more than one tortoise. The second tortoise is named Patricia!
+
+   .. sourcecode:: js
+      :linenos:
+
+      let tortoiseTwo = {
+          species: "Galapagos Tortoise",
+          name: "Patricia",
+          weight: 800,
+          age: 85,
+          diet: ["pumpkins", "lettuce", "cabbage"],
+          sign: function() {
+              return this.name + " is a " + this.species;
+          }
+       };
+
+   Because Pete and Patricia are members of the same species, are the same age, and have the same diet, you might notice that many of their properties are equal, but some are not.
+   Pete weighs more than Patricia and of course, they have different names!
+
+   For this example, we will only keep the ``species`` and ``diet`` properties.
+
+   .. sourcecode:: js
+      :linenos:
+
+      let tortoiseOne = {
+          species: "Galapagos Tortoise",
+          diet: ["pumpkins", "lettuce", "cabbage"]
+       };
+
+       let tortoiseTwo = {
+          species: "Galapagos Tortoise",
+          diet: ["pumpkins", "lettuce", "cabbage"],
+       };
+
+       console.log(tortoiseOne === tortoiseTwo);
+
+   **Console Output**
+
+   ::
+
+      False
```

Comment:
This might be too picky, but JavaScript expresses ``true`` and ``false`` as lowercase words.

Reply:
Autocorrect got me! Thanks!